storage/txcache/txMeasures.go

```diff
@@ -0,0 +1,27 @@
+package txcache
+
+import "github.com/ElrondNetwork/elrond-go/data"
+
+const estimatedSizeOfBoundedTxFields = uint32(128)
+
+// estimateTxSize returns an approximation
+func estimateTxSize(tx data.TransactionHandler) int64 {
+	sizeOfData := int64(len(tx.GetData()))
+	return int64(estimatedSizeOfBoundedTxFields) + sizeOfData
+}
+
+// estimateTxGas returns an approximation for the necessary computation units (gas units)
+func estimateTxGas(tx data.TransactionHandler) int64 {
```

Comment:
Why this has to return int64? With uint64 you can drop a cast

Reply:
Fixed. But I have to implement a new atomic counter of type `uint64`.